frontend/public/components/monitoring/dashboards/custom-time-range-modal.tsx

```diff
@@ -56,8 +66,7 @@ const CustomTimeRangeModal = ({ cancel, close, activePerspective }: CustomTimeRa
     endTime ? toISOTimeString(new Date(endTime)) : '23:59',
   );
 
-  const submit: React.FormEventHandler<HTMLFormElement> = (e) => {
-    e.preventDefault();
+  const submit: React.FormEventHandler<HTMLButtonElement> = () => {
```

Comment:
As the form is not used anymore then the type should be React.MouseEvent<HTMLButtonElement>. It can even be dropped  as we are not using the event.
```suggestion
  const submit: React.MouseEventHandler<HTMLButtonElement> = () => {
```

Reply:
👍